src/server/scripts/Northrend/IcecrownCitadel/boss_lady_deathwhisper.cpp

```diff
@@ -1002,6 +1004,37 @@ class spell_deathwhisper_summon_spirits : public SpellScript
     }
 };
 
+// 70674 - Vampiric Might
+class spell_deathwhisper_vampiric_might : public AuraScript
+{
+    PrepareAuraScript(spell_deathwhisper_vampiric_might);
+
+    bool Validate(SpellInfo const* /*spell*/) override
+    {
+        return ValidateSpellInfo({ SPELL_VAMPIRIC_MIGHT_PROC });
+    }
+
+    void HandleProc(AuraEffect const* aurEff, ProcEventInfo& eventInfo)
+    {
+        PreventDefaultAction();
+        DamageInfo* damageInfo = eventInfo.GetDamageInfo();
+        if (!damageInfo || !damageInfo->GetDamage())
+            return;
+
+        Unit* target = GetTarget();
+        uint32 damage = damageInfo->GetDamage();
+        AddPct(damage, aurEff->GetAmount());
```

Comment:
wont this make it heal for 400% instead of 300%?

Reply:
yeap, i dont saw it